plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -106,7 +106,10 @@ public class TestDeltaLakeBasic
             new ResourceTable("uniform_iceberg_v2", "databricks143/uniform_iceberg_v2"),
             new ResourceTable("unsupported_writer_feature", "deltalake/unsupported_writer_feature"),
             new ResourceTable("unsupported_writer_version", "deltalake/unsupported_writer_version"),
-            new ResourceTable("variant", "databricks153/variant"));
+            new ResourceTable("variant", "databricks153/variant"),
+            new ResourceTable("test_db_type_widening", "databricks153/type_widening"),
+            new ResourceTable("test_db_partition_type_widening", "databricks153/type_widening_partition"),
+            new ResourceTable("test_db_type_widening_nested", "databricks153/type_widening_nested"));
```

Comment:
Remove `test_db_` prefix for consistency with other table names. No need to recreate tables. Just updating the name here and README.md should be fine.